recipe/meta.yaml

```diff
@@ -24,12 +25,13 @@ requirements:
     - wheel
 
   run:
-    - python
+    - python >=3.6
 
 test:
   requires:
     - pyqt
     - pip
+    - python <3.10
   imports:
```

Comment:
We should add imports, see https://github.com/conda-forge/qtpy-feedstock/blob/7681e9d1466fe4580910fc9653faf5b2fddabdc2/recipe/meta.yaml#L31
Or maybe the scripts below replace imports